### PR TITLE
Dockerize application

### PR DESCRIPTION
**Pull request by JakobNH**
## Dockerizing the application

We created a compose.yaml file which runs on docker compose up, with two different docker files, one for backend and one for frontend. It still runs towards the local database. The readme is updated with a guide on how to run it in docker.

### CODE REVIEW

File: backend/src/main/resources/application.conf [hunk 1]
```diff
@@ -5,9 +5,4 @@ ktor {
     deployment {
         port = 8080
     }
-    database {
-        url = "jdbc:postgresql://localhost:5432/kontrollere"
-        user = "hein"
-        password = "hein"
-    }
 }
```
Comment (andersstandal) [inline backend/src/main/resources/application.conf:8]: Er denne nå unødvendig, eller må man fortsatt legge den til for lokal kjøring?

Comment (JakobNH) [reply backend/src/main/resources/application.conf:8]: Nei, de blir satt i ConfigureDatabase.kt om man skal kjøre lokalt nå

**Review by andersstandal — APPROVED** [review]
SBUFM (ser bra ut for meg)

**Review by SondreKanstad — APPROVED** [review]
LGTM!

**Pull request merged**